.github/workflows/bot-pr-comment.yml

```
@@ -14,7 +14,7 @@ jobs:
     - name: Add appropriate labels
       env:
         GITHUB_TOKEN: ${{ secrets.BOT_TOKEN }}
-      run: cp .github/label.sh /tmp && /tmp/label.sh
+      run: cp .github/label.sh /tmp && chmod +x /tmp/label.sh && /tmp/label.sh
```

This applies to all changes below as well:
If you want to make absolutely sure that the mode is preserved while copying, you can use `cp --preserve` or `cp --preserve=mode` (removing the need for `chmod`):
```suggestion
      run: cp --preserve .github/label.sh /tmp && /tmp/label.sh
```